app/commands.py

```diff
@@ -799,6 +799,24 @@ def update_templates():
         data = json.load(f)
         for d in data:
             _update_template(d["id"], d["name"], d["type"], d["content"], d["subject"])
+    _clear_templates_from_cache()
+
+
+def _clear_templates_from_cache():
+    # When we update-templates in the db, we need to make sure to delete them
+    # from redis, otherwise the old versions will stick around forever.
+    CACHE_KEYS = [
+                    "service-????????-????-????-????-????????????-templates",
+                    "service-????????-????-????-????-????????????-template-????????-????-????-????-????????????-version-*",  # noqa
+                    "service-????????-????-????-????-????????????-template-????????-????-????-????-????????????-versions",  # noqa
+                ]
+
+
+    count1 = redis_store.delete_by_pattern(CACHE_KEYS[1])
```

Comment:
Do we need to assign a variable here and is it used for anything, or can we just make the call to `redis_store`?

Reply:
That was for debugging, removed it.